% !TeX encoding = UTF-8
% Do not touch the below 70 lines
\documentclass[11pt, a4paper, onecolumn, oneside]{report}

\usepackage{mathptmx}
\usepackage[T1]{fontenc}
\usepackage[utf8]{inputenc}

\RequirePackage[top=3cm, bottom=1in, left=1in, right=1in]{geometry}
\linespread{1.3}

\usepackage{titlesec}
\usepackage{amsmath}
\usepackage{amssymb}
\usepackage{mathtools}
\usepackage{enumerate}
\usepackage{bbm}
\usepackage{algorithm}
\usepackage{algorithmic}
\usepackage{epsfig}
\usepackage{color}
\usepackage{graphicx}
\usepackage{caption}
\usepackage{subcaption}
\usepackage{cases}
\usepackage{url}
\usepackage{cite}
\usepackage{fancyhdr}
\usepackage{tocloft}
\usepackage{pdfpages}

\usepackage{acro}
\usepackage[notocbib]{apacite}
\usepackage{kotex}
\usepackage{microtype}
\setcounter{secnumdepth}{5}
\setcounter{tocdepth}{5}

\input{Documents/abbreviations.tex}

\renewcommand\cftsecafterpnum{\vskip15pt}
\renewcommand\cftsubsecafterpnum{\vskip15pt}
\renewcommand\cftfigafterpnum{\vskip15pt}
\renewcommand{\thesection}{\arabic{section}}
\renewcommand{\thesubsection}{\arabic{section}.\arabic{subsection}}
\renewcommand{\thesubsubsection}{\arabic{section}.\arabic{subsection}.\arabic{subsubsection}}
\renewcommand{\contentsname}{\hfill\bfseries\Large Contents\hfill}
\renewcommand{\listfigurename}{\hfill\bfseries\Large List of Figures\hfill}
\renewcommand{\listtablename}{\hfill\bfseries\Large List of Tables\hfill}
\renewcommand{\thefigure}{\arabic{figure}}
\newcommand{\qed}{\hfill\blacksquare}
\renewcommand{\bibname}{\hfill\bfseries\Large References \hfill\hfill}
\renewcommand{\abstractname}{\bfseries\Large Abstract \hfill\hfill}

\newcounter{lemma}
\newcounter{proposition}
\newcounter{theorem}
\newtheorem{lemma}{\bf Lemma}
\newtheorem{proposition}{\bf Proposition}
\newtheorem{theorem}{\bf Theorem}
\newtheorem{proof}{\bf Proof}

\newcommand{\HIGH}[1]{{\textcolor{blue}{#1}}}
%\renewcommand{\baselinestretch}{1.5}

\DeclareMathOperator*{\argmax}{arg\,max}

\fancyhf{}
\renewcommand{\headrulewidth}{0pt}
\cfoot{\thepage}
\pagestyle{fancy}
%\pagenumbering{gobble}

% Do not touch the above 70 lines

\begin{document}
% Front cover
    \begin{center}
    \LARGE Doctoral Thesis

    \vspace{3cm}
    \huge \input{Documents/Title.txt}

    \vfill

    \LARGE \input{Documents/Name.txt}

    \vspace{2cm}

    \LARGE \input{Documents/Department.txt}

    \vspace{2cm}

    \LARGE \input{Documents/UNIST.txt}
    \vspace{2cm}

    \LARGE \the\year{}

    \end{center}
    \thispagestyle{empty}
    \clearpage

% Title page
    \begin{center}
    \hbox{ }

    \hbox{ }

    \huge \input{Documents/Title.txt}

    \vspace{5cm}

    \LARGE \input{Documents/Name.txt}

    \vspace{6cm}

    \LARGE \input{Documents/Department.txt}

    \vspace{2cm}

    \LARGE \input{Documents/UNIST.txt}

    \end{center}
    \thispagestyle{empty}
    \clearpage

% Thesis approval
% Add the approval doc signed by your advisor in a PDF file
% Put your pdf with the filename below, and uncomment it.
\includepdf[fitpaper= true, pages=-]{Documents/example.pdf}

% [Confirmation of thesis approval]
% add the certificate signed by your committee in a PDF file
% Put your pdf with the filename below, and uncomment it.
\includepdf[fitpaper= true, pages=-]{Documents/example.pdf}

% Abstract
    \begin{abstract}
        The microbiome refers to the complex community of microorganisms, including bacteria, viruses, fungi, and other microbes, that inhabit various environment within living organisms \cite{microbiome-1, microbiome-3}. In humans, the microbiome plays a crucial role in maintaining health \cite{microbiome-2}, influencing processes such as digestion \cite{microbiome-digestion-1}, immune response \cite{microbiome-immune-1, microbiome-immune-2, microbiome-immune-3}, and even mental health \cite{microbiome-mental-1, microbiome-mental-2, microbiome-mental-3}. These microbial communities are not static nor constant, but rather dynamic ecosystem that interacts with their host and respond to environmental changes. Recent studies have revealed that imbalances in the microbiome, known as dysbiosis, can contribute to a wide range of diseases, including obesity \cite{microbiome-obesity-1, microbiome-obesity-2, microbiome-obesity-3}, diabetes \cite{microbiome-diabetes-1, microbiome-diabetes-2, microbiome-diabetes-3}, infections \cite{microbiome-infection-1, microbiome-infection-2}, inflammatory conditions \cite{microbiome-inflammation-1, microbiome-inflammation-2, microbiome-inflammation-3}, and cancers \cite{microbiome-cancer-1, microbiome-cancer-2, microbiome-cancer-3, microbiome-cancer-4}. Thus, understanding the composition of the human microbiomes is essential for developing new therapeutic approaches that target these microbial populations to promote health and prevent diseases.

        (PTB)

        (Periodontitis)

        (Lung)

        (Conclusion)

        \vfill
        \noindent\rule{\linewidth}{0.4pt}
        \textbf{This doctoral dissertation is an addition based on the following papers that the author has already published}:
        \begin{itemize}
            \item \input{Documents/PTB.txt} \nocite{PTB-JW-1}
        \end{itemize}
    \end{abstract}
    \clearpage

% Do not touch the below 20 lines
\hbox{ }
\thispagestyle{empty}
\clearpage

%%% Table of Contents
\tableofcontents{}
\thispagestyle{empty}
\vfill
\clearpage

%%% List of Figures
\listoffigures{}
\thispagestyle{empty}
\clearpage

%%% List of Tables
\listoftables{}
\thispagestyle{empty}
\clearpage

%%% reset page numbering
\setcounter{page}{1}
%  Do not touch the above 20 lines

    \section*{\centering List of Abbreviations}
    \printacronyms[display=all, heading=none]
    \newpage

    \section{Introduction}
        (Diversity indices)

        (Classification evaluation)
    \newpage

    \section{Predicting preterm birth using machine learning techniques in oral microbiome}
        \label{section:PTB}

        \textbf{This section includes the published contents}: \\
        \input{Documents/PTB.txt} \nocite{PTB-JW-1}

        \subsection{Introduction}
            Preterm birth (PTB), characterized by the delivery of neonates prior to 37 weeks of gestation, is one of the major cause to neonatal mortality and morbidity \cite{PTB-rate-1}. Multiple pregnancies including twins, short cervical length, and infection on genitourinary tract are known risk factor for PTB \cite{PTB-cause-1}. Nevertheless, the extent to which these aspects affect birth outcomes is still up for debate. Henceforth, strategies to boost gestation and enhance delivery outcomes can be more conveniently implemented when pregnant women at high risk of PTB are identified early \cite{PTB-care-1}.

            Prediction models that can be utilized as a foundation for intervention methods still have an unacceptable amount of classification evaluations, including accuracy, sensitivity, and specificity, despite a great awareness of the risk factors that trigger PTB \cite{PTB-prediction-1}. Several attempts have been made to predict PTB through integrating data such as human microbiome composition, inflammatory markers, and prior clinical data with predictive machine learning methods \cite{PTB-prediction-2}. Because it is affordable and straightforward to use, fetal fibronectin is commonly used in medical applications. However, with a sensitivity of only 56\% that merely similar to random prediction, it has a low classification evaluation \cite{PTB-prediction-3}. Due to the difficulty and imprecision of the method in general, as well as the requirement for a qualified specialist cervical length measuring is also restricted \cite{PTB-prediction-4}.

            Preterm prelabor rupture of membranes (PROM) brought on by gestational inflammation and infection contribute to about 70\% of PTB cases \cite{PTB-prediction-5}. Nevertheless, as antibiotics and anti-inflammatory therapeutic strategies were ineffective to decrease PTB occurrence rates, the pathology of PTB has not been entirely elucidated by inflammatory and infectious pathways \cite{PTB-mechanism-1}. Recent researches on maternal microbiomes were beginning to examine unidentified connections of PTB as a consequence of developmental processes in molecular biological technology \cite{PTB-mechanism-2}.

            However, as anti-inflammatory and antibiotic therapies were insufficient to lower PTB occurrence rates, infectious and inflammatory processes are insufficient to exhaustively clarify the pathogenesis and pathophysiology of PTB. It has been hypothesized that the microbiota linked to PTB originate from either a hematogenous pathway or the female genitourinary tract increasing through the vagina and/or cervix. \cite{PTB-mechanism-3}. Vaginal microbiome compositions have been found in women who eventually acquire PTB, and recent studies have tried to predict PTB risk using cervicovaginal fluid \cite{PTB-mechanism-4}. Even though previous investigation have confirmed the potential relationships between the vaginal microbiome compositions and PTB, these studies are only able to clarify an upward trajectory.

            Multiple unfavorable birth outcomes, including PROM and PTB, have been linked to periodontitis as an independence risk factor, according to numerous epidemiological researches \cite{PTB-mechanism-5}. It is expected that the oral microbiome will be able to explain additional hematogenous pathways in light of these precedents; however, the oral microbiome composition of fetuses is limited understood.

            Hence, in order to identify the salivary microbiome linked to PTB and to establish a machine learning prediction model of PTB determined by oral microbiome compositions, this study examined the salivary microbiome compositions of PTB study participants with a full-term birth (FTB) study participants.
        \newpage

        \subsection{Materials and methods}
            \subsubsection{Study design and study participants}
                Between 2019 and 2021, singleton pregnant women who received treatment to Jeonbuk National University Hospital for childbirth were the participants of this study. This study was conducted according to the Declaration of Helsinki \cite{Helsinki-1}. The Institutional Review Board authorized this study (IRB file No. 2019-01-024). Participants who were admitted for elective cesarean sections (C-sections) or induction births, as well as those who had written informed consent obtained with premature labor or PROM, were eligible.

            \subsubsection{Clinical data collection and grouping}
                Questionnaires and electronic medical records were implemented to gather information on both previous and current pregnancy outcomes. These clinical data included known risk factors, namely maternal age at delivery, diabetes mellitus, hypertension, overweight, C-section, PROM, and history of PTB, along with demographic neonatal factors, such as gestational week on birth, weight, and sex.

            \subsubsection{Salivary microbiome sample collection}
                Salivary microbiome samples were collected 24 hours before to delivery using mouthwash. The standard methods of sterilizing were performed. Medical experts oversaw each stage of the sample collecting procedure. Participants received instruction not to eat, drink, or brush their teeth for 30 minutes before sampling salivary microbiome. Saliva samples were gathered by washing the mouth for 30 seconds with 12 mL of a mouthwash solution (E-zen Gargle, JN Pharm, Pyeongtaek, Korea). The samples were tagged with the anonymous ID for each participant and kept at 4 \textcelsius until they underwent further processing. Genomic DNA was extracted using an ExgeneTM Clinic SV kit (GeneAll Biotechnology, Seoul, Korea) following with the manufacturer instructions and store at -20 \textcelsius.

            \subsubsection{16s rRNA gene sequencing}
                Salivary microbiome samples were transported to \input{Documents/Department.txt} of \input{Documents/UNIST.txt}. 16S rRNA sequencing was then carried out using a commissioned Illumina MiSeq Reagent Kit v3 (Illumina, San Diego, CA, USA). Library methods were utilized to amplify the V3-V4 areas. 300 base-pair paired end reads were produced by sequencing the pooled library using a v3 600 cycle chemistry after the samples had been diluted to a final concentration of 6 pM with a 20\% PhiX control.

            \subsubsection{Bioinformatics analysis}
                The independent $t$-test was utilized to evaluate the differences of continuous values between from the PTB participants than the FTB participants; $\chi$-squared test was applied to decide statistical differences of categorical values. Clinical measurement comparisons were conducted using SPSS (version 20.0) \cite{SPSS-1}. At $p < 0.05$, statistical significance was taken into consideration.

                QIIME2 (version 2022.2) was implemented to import 16S rRNA gene sequences from salivary microbiome samples of study participants for additional bioinformatics processing \cite{QIIME2-1}. DADA2 was used to verify the qualities of raw sequences \cite{DADA2-1}. The remain sequences were clustered into amplicon sequence variants (ASVs). Diversity indices, namely Faith PD for alpha diversity index \cite{FaithPD-1} and Hamming distance for beta diversity index \cite{Hamming-1}, were calculated. Mann–Whitney–Wilcoxon U-test and PERMANOVA multivariate test were evaluated for measuring statistical significance \cite{PERMANOVA-1, PERMANOVA-2}.

                Taxonomic assignment were implemented with HOMD (version 15.22) \cite{HOMD-1}. Afterward, DESeq2 was implemented to identify differentially abundant taxa (DAT) that could distinguish between salivary microbiome from PTB and FTB participants \cite{DESeq2-1}. Taxa with $| \log _2 \textrm{FoldChange} |$ $>$ 1 and $p < 0.05$ were considered as statistically significant.

                The taxa for predicting PTB using salivary microbiome data were determined using a random forest classifier \cite{RF-1}. Through stratified $k$-fold cross-validation ($k=5$) that preserves the existence rate of PTB and FTB participants, consistency and trust worthy classification were ensured \cite{Kfold-1}.
        \newpage

        \subsection{Results}
            \subsubsection{Overview of clinical information}
                In the beginning, 69 volunteer mothers were recruited for this study. However, due to insufficient clinical information or twin pregnancies, 10 participants were excluded from the study participants. Demographic and clinical information of the study participants are displayed in Table \ref{tab:PTB-clinical}. Because PROM is one of the leading factors of PTB, it was prevalent in the PTB group than the FTB group. Other maternal clinical factors did not significantly differ between the FTB and PTB groups. There were no cases in both groups that had a history of simultaneous periodontal disease or cigarette smoking.

            \subsubsection{Comparison of salivary microbiomes composition}
                The salivary microbiome composition was composed of 13953804 sequences from 59 study participants, with 102305.95$\pm$19095.60 and 64823.41$\pm$15841.65 (mean$\pm$SD) reads/sample before and following the quality-check stage, accordingly. There was not a significant distinction between the PTB and FTB groups with regard to on alpha diversity nor beta diversity metrics (Figure \ref{fig:PTB-diversity}).

                DESeq2 was used to select 32 DAT that distinguish between the PTB and FTB groups out of the 465 species that were examined \cite{DESeq2-1}: 26 FTB-enriched DAT and six PTB-enriched DAT. Seven PROM-related DAT were removed from these 32 PTB-related DAT to lessen the confounding effect of PROM (Figure \ref{fig:PTB-volcano}). Therefore, there were a total of 25 PTB-related DAT: 22 FTB-enriched DAT and three PTB-enriched DAT (Figure \ref{fig:PTB-DAT}).

                A significant negative correlation was found using Pearson correlation analysis between GW and differences between PTB-enriched DAT and FTB-enriched DAT ($r = -0.542$ and $p = 7.8\textrm{e}-6$; Figure \ref{fig:PTB-volcano}).

            \subsubsection{Random forest classification to predict PTB risk}
                To classify PTB according to DAT, random forest classifiers were constructed. The nine most significant DAT were used to obtain the best BA (0.765$\pm$0.071; Figure \ref{fig:PTB-ML}a). Moreover, random forest classification model determined each DAT's importance (Figure \ref{fig:PTB-ML}b). We conducted a validation procedure on nine twin pregnancies that were excluded in the initial study design in order to confirm the reliability and dependability of our random forest-based PTB prediction model (Figure \ref{fig:PTB-validation}). Comparable to the PTB prediction model on the 59 initial singleton study participants, the validation classification on PTB risk of these twin participants have an accuracy of 87.5\%.

            \begin{table}[p]
                \centering

                \caption[Standard clinical characteristics of study participants]{\textbf{Standard clinical characteristics of study participants}. Continuous variable are displayed as Mean$\pm$standard deviation. Categorical variable are displayed as count (proprotion). Continuous variable for independent $t$-test. Categorical variable for Pearson’s $\chi$-square test.}
                \label{tab:PTB-clinical}
            \end{table}
            \clearpage

            \begin{figure}[p]
                \centering
                \includegraphics[width=12 cm]{Figures/PTB/Fig1-DAT.pdf}
                \caption[DAT volcano plot]{\textbf{DAT volcano plot}. PTB-enriched DAT shown as red dots and FTB-enriched DAT shown as green dots. Taxa with $| \log _2 \textrm{FoldChange} |$ $>$ 1 and $\textrm{adjusted-} p < 0.05$ are considered as significant different.}
                \label{fig:PTB-DAT}
            \end{figure}
            \clearpage

            \begin{figure}[p]
                \centering
                \includegraphics[width=15 cm]{Figures/PTB/Fig2-Composition.pdf}
                \caption[Oral microbiome compositions over DAT]{\textbf{Oral microbiome compositions over DAT}. \textbf{(a)} Proportions of DAT of study subjects. Samples are sorted by difference between PTB-enriched DAT proportion and FTB-enriched DAT proportion. GA of samples are shown, matched with order of upper panel. PTB: red bar, FTB: green bar. PROM: arrow head. \textbf{(b)} Correlation plot with GA and difference between PTB-enriched DAT proportion and FTB-enriched DAT proportion. Pearson correlation shows strong negative coefficient ($r = -0.542$ and $p = 7.8\textrm{e}-06$) between GA and difference between PTB-enriched DAT proportion and FTB-enriched DAT proportion.}
                \label{fig:PTB-composition}
            \end{figure}
            \clearpage

            \begin{figure}[p]
                \centering
                \includegraphics[width=15 cm]{Figures/PTB/Fig3-ML.pdf}
                \caption[Machine learning evaluations over DAT]{\textbf{Machine learning evaluations over DAT}. \textbf{(a)} Machine learning evaluation upon number of features (DAT). Random Forest classifier has the best balanced accuracy (mean $\pm$ standard deviation.; 0.765 $\pm$ 0.071) with the nine most important DAT. \textbf{(b)} Importance of DAT is shown. Note that $0 \le \textrm{importance} \le 1$, and $\sum \textrm{importance} =$ 1. The 20 most important DAT that give the best balanced accuracy is marked with blue.}
                \label{fig:PTB-ML}
            \end{figure}
            \clearpage

            \begin{figure}[p]
                \centering
                \includegraphics[width=15 cm]{Figures/PTB/FigS1-Diversity.pdf}
                \caption[Diversity indices]{\textbf{Diversity indices}. \textbf{(a)} Alpha diversity index (Faith PD). Mann-Whitney-Wilcoxon test did not find statistically significant difference between the PTB group and the FTB group. Mean values are marked with star-point. \textbf{(b)} t-SNE plot with beta diversity index (Hamming distance). PERMANOVA multivariate test did not find statistically significant difference between the PTB group and the FTB group.}
                \label{fig:PTB-diversity}
            \end{figure}
            \clearpage

            \begin{figure}[p]
                \centering
                \includegraphics[width=12 cm]{Figures/PTB/FigS2-PROM.pdf}
                \caption[PROM-related DAT volcano plot]{\textbf{PROM-related DAT volcano plot}. This is a subgroup analysis between 12 participants in PTB group with PROM and 18 participants in PTB group without PROM, 42 PROM-related DAT were selected between these two groups. Out of these 42 PROM-related DAT, only 7 DAT overlapped with PTB-related DAT, as it indicated by the bold marking. Volcano plot shows PROM-related DAT, with 12 PROM-underrepresented DAT shown as blue dots and 30 PROM-overrepresented DAT shown as red dots. Taxa with $| \log _2 \textrm{FoldChange} |$ $>$ 1 and adjusted $p < 0.05$ are considered as significant different.}
                \label{fig:PTB-volcano}
            \end{figure}
            \clearpage

            \begin{figure}[p]
                \centering
                \includegraphics[width=12 cm]{Figures/PTB/FigS3-validation}
                \caption[Heatmap plot of PTB classification with validation data]{\textbf{Heatmap plot of PTB classification with validation data}. A validation test on 9 twin pregnancies that had excluded from the paper was conducted. They consist of 8 PTB subjects and 1 FTB subject. As twin pregnancies have a 7-10 times higher PTB rate than singleton pregnancies, resulting in a majority of the test data falling into the PTB group. The machine learning classifications have 87.5\% accuracy, comparable to the machine learning classification on the singleton study subjects (Mean $\pm$ standard deviation. 0.714 $\pm$ 0.061)}
                \label{fig:PTB-validation}
            \end{figure}
            \clearpage
        \newpage

        \subsection{Discussion}
            In this study, we developed a method for predicting PTB based on random forest classifier using salivary microbiome compositions. Recently, several sporadic reports have suggested a bidirectional relationship between oral microbiome and pregnancy \cite{PTB-mechanism-3}. However, prenatal oral microbiome is not well understood yet. Some research has shown that oral microbial dysbiosis combined with gingival inflammation can lead to adverse pregnancy  outcomes, including low birth weight, PTB, pre-eclampsia, and miscarriages \cite{PTB-mechanism-6}. Nevertheless, these results have been inconsistent due to methodologies employed in studies that only target known pathogens.

            \textit{Fusobacterium nucleatum} is the most prevalent oral microbiome studied \cite{Fusobacterium-1, Fusobacterium-2, Fusobacterium-3}. \textit{Fusobacterium nucleatum} is a Gramnegative, anaerobic, filamentous oral microbiome. It is considered one of the most abundant species in the oral microbiome. It can also be isolated from vaginal microbiome \cite{PTB-mechanism-7, PTB-mechanism-8}. Intra-amniotic \textit{Fusobacterium nucleatu}m infection leading to PTB has been reported in human and animal studies \cite{Fusobacterium-4}. Other studies have shown that other oral pathogens including \textit{Porphyromonas gingivalis} and intrauterine \textit{Bergeyella} spp. can be isolated from the placenta of women who deliver prematurely \cite{Porphyromonas-1, Porphyromonas-2}. In the present study, although \textit{Bergeyella} spp. was overrepresented in the PTB group with PROM, it was excluded in the finally selected 25 DAT. Furthermore, \textit{Campylobacter gracilis} was one of the FTB-enriched DAT that can aid colonization by periodontal pathogens including \textit{Porpyhromonas gingivalis} in subgingival microbiome \cite{PTB-mechanism-9}. \textit{Lactobacillus gasseri }was also one of the FTB-enriched DAT. It is known that \textit{Lactobacillus gasseri} in vaginal microbiome can decrease early PTB risk \cite{PTB-mechanism-10, PTB-mechanism-11}.

            We found that decisive species differentiating between two groups were mainly abundant in the FTB group, with DAT consisting of 22 FTB-enriched DAT and three PTB-enriched DAT. We hypothesize that deficiency of species having a protective impact might have triggered the pathophysiology of PTB. Two different mechanisms have been proposed to explain the relationship between unhealthy microbiota composition and adverse pregnancy outcomes. The first mechanism proposed that periodontal bacteria originating in the gingival biofilm could translocate from the unhealthy oral cavity and cross the placenta, reach the intra-amniotic fluid and fetal circulation and directly affect the fetoplacental unit, resulting in bacteremia \cite{Periodontitis-1}. The second mechanisms proposed that systemic dissemination of endotoxins and/or inflammatory mediators derived from periodontal plaque and  secreted by the subgingival inflammatory site could be carried to the fetoplacental unit \cite{PTB-mechanism-12, PTB-mechanism-13}. Although certain microbiota has the same species, their subgroups can have both positive and negative influences on pregnancy outcomes. Following this line of thought, we believe that composition or dysbiosis of the oral microbiome is more important than the presence of specific microbiota.

            It is worth nothing that microbial changes occurring during pregnancy might be nature consequences of a healthy pregnancy. Three reasons can explain the susceptibility to oral diseases such as periodontitis during pregnancy. These diseases are common in pregnant women due to hormonal driven hyper-reactivity of the gingiva to bacteria in the subgingival biofilm. Other factors that increase the risk of poor oral health during pregnancy include changes in dietary habits (frequent snacking or increased consumption of carbohydrate-rich foods), stomach acids from nausea and vomiting that contribute to the breakdown of tooth enamel, and a decreased likelihood of seeking dental care during pregnancy. We plan to implement pathway analyses to investigate direct link between the microbiome and PTB.

            Even though there was limited power resulting from a small number of participants and restricted validation sample size, our study verified that oral microbiota might provide potential biomarkers for predicting pregnancy complications using machine learning methods including random forest classification. Additionally, the fact that the entire microbiome was not analyzed was a limitation of this study because our analysis only used relative values measured by 16S rRNA sequencing, not 16S meta-genome sequencing. We did not investigate other factors could impact the oral microbiome, such as participant’s diet and socioeconomic status.

            Despite these limitations, this prospective study demonstrated the potential of a PTB prediction model using oral microbiome in mouthwash. Further multi-center and larger-scale studies are needed to confirm out results before applying techniques developed in this study in the clinical field.
        \newpage

    \section{Prediction model for periodontitis severity based on the salivary microbiomes}
        \label{section:Periodontitis}

        \textbf{This section includes the published contents:} \\

        \subsection{Introduction}
        \newpage

        \subsection{Materials and methods}
            \subsubsection{Study subjects and clinical procedure}
            \subsubsection{Bioinformatics analysis}
        \newpage

        \subsection{Results}
            \subsubsection{Summary of study subjects and sequencing data}
            \subsubsection{Diversity indices reveal differences in diversity among the periodontal statuses}
            \subsubsection{Differentially abundant taxa among healthy and different periodontitis stages and their correlation}
            \subsubsection{Classification of periodontitis stages by random forest models}

            \begin{table}[p]
                \centering

                \caption[Clinical characteristics of the study subjects]{\textbf{Clinical characteristics of the study subjects}. Significant differences were assessed using the Kruskal-Wallis test. NA: Not applicable.}
                \label{tab:Periodontitis-clinical}
            \end{table}
            \clearpage

            \begin{figure}[p]
                \centering
                \includegraphics[width=15 cm]{Figures/Periodontitis/Figure_1.pdf}
                \caption[Diversity indices]{\textbf{Diversity indices}. Comparisons of salivary microbiomes among healthy controls and patients with various periodontitis stages. Alpha-diversity indices \textbf{(a-e)} indicate that healthy controls have increased heterogeneity than periodontitis stages as measured by: \textbf{(a)} ace \textbf{(b)} chao1 \textbf{(c)} Fisher alpha \textbf{(d)} Margalef, and \textbf{(e)} observed ASVs. \textbf{(f)} The beta-diversity index (weighted UniFrac) was visualized using a tSNE-transformed plot. The confidence ellipses are shown to display the distribution of each periodontitis stage. The distance to each stage demonstrated that each periodontitis stage was distinguished from the other periodontitis stages: \textbf{(g)} distance to Healthy \textbf{(h)} distance to Stage I \textbf{(i)} distance to Stage II, and \textbf{(j)} distance to Stage III. Statistical significance determined by the Mann-Whitney U-test (MWU) test: $p \le 0.01$ (**) and $p \le 0.0001$ (****).}
                \label{fig:Periodontitis-diveristy}
            \end{figure}
            \clearpage

            \begin{figure}[p]
                \centering
                \includegraphics[width=15 cm]{Figures/Periodontitis/Figure_2.pdf}
                \caption[Differentially abundant taxa]{\textbf{Differentially abundant taxa}. Differentially abundant taxa (DAT) that were identified by ANCOM. \textbf{(a)} Heatmap of clustered DAT with similar distribution among subjects. Group 1, Group 2, and Group 3 are marked in red, black, and green, respectively. \textbf{(b)} Box plots showing the proportions of DAT. Taxa were sorted by their importance according to ANCOM.}
                \label{fig:Periodontitis-DAT}
            \end{figure}
            \clearpage

            \begin{figure}[p]
                \centering
                \includegraphics[width=12 cm]{Figures/Periodontitis/Figure_3.pdf}
                \caption[Correlation heatmap]{\textbf{Correlation heatmap}. Pearson’s correlations between differentially abundant taxa in healthy status and periodontitis stages. Statistical significance was determined by strong correlation, i.e., $| \textrm{coefficient} | \ge 0.5$ (*).}
                \label{fig:Periodontitis-correlation}
            \end{figure}
            \clearpage

            \begin{figure}[p]
                \centering
                \includegraphics[width=15 cm]{Figures/Periodontitis/Figure_4.pdf}
                \caption[Random forest classification metrics]{\textbf{Random forest classification metrics}. The classification metrics in the random forest classifications were as follows: accuracy (ACC), area-under-curve (AUC), balanced accuracy (BA), F1 score (F1), precision (PRE), sensitivity (SEN), and specificity (SPE). Every classification metric ranges from [0, 1], with higher values indicating better performance. \textbf{(a)} Classification performance for healthy vs. stage I vs. stage II vs. stage III. \textbf{(b)} Receiver-operating characteristics (ROC) curve for the highest BA of (a). \textbf{(c)} Classification performance for healthy vs. stage I. \textbf{(d)} ROC curve on the highest BA of (c). (\textbf{e)} Classification performance for healthy vs. stage I vs. stages II/III. \textbf{(f)} ROC curve for the highest BA of (e). \textbf{(g)} Classification performance for healthy vs. stages I/II/III. \textbf{(h)} ROC curve for the highest BA of (h).}
                \label{fig:Periodontitis-ML}
            \end{figure}
            \clearpage

            \begin{figure}[p]
                \centering
                \includegraphics[width=12 cm]{Figures/Periodontitis/Figure_S1.pdf}
                \caption[Rarefaction curves for alpha diversity indices]{\textbf{Rarefaction curves for alpha diversity indices}. Rarefaction of \textbf{(a)} chao1 \textbf{(b)} margalef, and \textbf{(c)} observed ASVs were generated to measure species richness and determine the sampling depth of each sample.}
                \label{fig:Periodontitis-rarefaction}
            \end{figure}
            \clearpage

            \begin{figure}[p]
                \centering
                \includegraphics[width=15 cm]{Figures/Periodontitis/Figure_S2.pdf}
                \caption[Absolute abundance of salivary bacterial taxa in the different periodontal statuses at the species level]{\textbf{Absolute abundance of salivary bacterial taxa in the different periodontal statuses at the species level}. Stacked bar plot of the absolute abundance of bacterial species for all samples \textbf{(a)} and the mean absolute abundance of bacterial species in the healthy, stage I, stage II, and stage III groups \textbf{(b)}.}
                \label{fig:Periodontitis-abundance}
            \end{figure}
            \clearpage

            \begin{figure}[p]
                \centering
                \includegraphics[width=12 cm]{Figures/Periodontitis/Figure_S3.pdf}
                \caption[Correlation plots for differentially abundant taxa]{\textbf{Correlation plots for differentially abundant taxa}. We selected the combinations of DAT with absolute Spearman correlation coefficients greater than 0.5. The color represents periodontal healthy periodontal statuses (green: healthy, cyan: stage I, yellow: stage II, and red: stage III).}
                \label{fig:Periodontitis-correlation2}
            \end{figure}
            \clearpage
        \newpage

        \subsection{Discussion}
        \newpage

    \section{Lung microbiome}
        \subsection{Introduction}
        \clearpage

        \subsection{Materials and methods}
        \clearpage

        \subsection{Results}
        \clearpage

        \subsubsection{Discussion}
        \clearpage
    \newpage

    \section{General conclusion and future perspective}
        \label{section:conclusion}
        \subsection{General conclusions}
            In conclusion, the research described in this doctoral dissertation was conducted to identify significant ...

            In the Section \ref{section:PTB}, I show that
        \newpage

        \subsection{Plan for future}
        \newpage

        \subsection{Future perspective}
        \newpage

% Reference
    \addcontentsline{toc}{section}{References}
    \bibliographystyle{apacite}
    \bibliography{references.bib}
    \clearpage

% Acknowledgements
    \addcontentsline{toc}{section}{Acknowledgments}
    \section*{\hfill \Large Acknowledgments \hfill}
        I would like to disclose my earnest appreciation for my advisor, Professor Semin Lee, who provided solicitous supervision and cherished opportunities throughout the course of my research. His advice and consultation encouraged me to become as a researcher and to receive all humility and gentleness. I am also grateful to all of my committee members, Professor AAA, Professor BBB, Professor CCC, and Professor DDD, for their critical and meaningful mentions and suggestions.

        I extend my deepest gratitude to my Lord, \textit{the Flying Spaghetti Monster}, His Noodly Appendage has guided me through the twist and turns of this academic journey. His presence, ever comforting and mysterious, has been a source of strength and humor during both highs and lows. In moments of doubt, I found solace in the belief that you were there, gently reminding me to keep faith in the process. His Holy Noodle has nourished my mind, and for that, I am truly overwhelmed. May His Holy Noodle continue to guide me in all my future endeavors. R'Amen.

        (Professors)

        I would like to extend my heartfelt gratitude to my colleagues of the Computational Biology Lab @ UNIST, whose collaboration, friendship, brotherhood, and support have been an invaluable part of my journey. Your willingness to share insights, engage in thoughtful discussions, and offer encouragement during the challenging moments of research has significantly shaped my academic experience. The camaraderie in Computational Biology Lab made even the most demanding days more enjoyable, and I am deeply grateful for the collaborative environment we created together. I appreciate you for standing by my side throughout this Ph.D. journey.

        I would like to express my heartfelt gratitude to my family, whose unwavering support has been the foundation of everything I have achieved. Your love, encouragement, and belief in me have sustained me through every challenge, and I could not have come this far without you. From your words of wisdom to your patience and understanding, each of you has played a vital role in helping me navigate this journey. The strength and comfort I have drawn from our family bond have been my greatest source of resilience. Your presence, both near and far, has filled my life with warmth and motivation. I am deeply grateful for your unconditional love and for always being there when I needed you the most. Thank you for being my constant source of strength and inspiration.

        I am incredibly pleased to my friends, especially my GSHS alumni (이망톡), for their unwavering support and encouragement throughout this journey. The bonds we formed back in our school days have only grown stronger over the years, and I am fortunate to have had such loyal and understanding friends by my side. Your constant words of motivation, and even moments of levity during stressful times have helped keep me grounded. Whether it was a late-night conversations, a shared laugh, or a simple message of reassurance, you all have played a vital role in keeping me focused and motivated. I am relieved for the ways you celebrated each small achievement with me and how you patiently listened to my worries. The memories of our shared past provided me with comfort and a sense of stability when the road ahead felt uncertain. I could not have reached this point without the love and friendship that you all have generously given. Each of your, in your unique way, has contributed to this dissertation, even if indirectly, and for that, I am forever beholden. I look forward to continuing our friendship as we all grow in our individual paths, knowing that the support we share is something truly special.

        I would like to express my sincere gratitude to the amazing members of my animal protection groups, DRDR (두루두루) and UNIMALS (유니멀스), whose dedication and compassion have been a constant source of motivation. Your unwavering commitment to improving the lives of animals has inspired me throughout this journey. I am also thankful for the beautiful cats we have cared for, whose presence brought both joy and purpose to our allegiance. Their playful spirits and gentle companionship served as daily reminders of why we continue to fight for animal rights. The bond we share, both with each other and with the animals we protect, has enriched my life in countless ways. I appreciate you all again for your support, dedication, and for being part of this meaningful cause.

        I would like to express my deepest gratitude to everyone I have had the honor of meeting throughout this journey. Your kindness, encouragement, and support have carried me through both the challenging and rewarding moments of my life. Whether through a kind word, thoughtful advice, or simply being there when I needed it most, your presence has made all the difference. I am incredibly fortunate to have received such generosity and warmth from those around me, and I do not take it for granted. Every act of kindness, no matter how big or small, has been a source of strength and motivation for me. To all my friends, colleagues, mentors, and beloved ones, thank you for your unwavering support. I am truly grateful for each of you, and your kindness has left an indelible mark on my journey.

        \begin{center}
            My Lord, \textit{the Flying Spaghetti Monster},\\
            give us grace to accept with serenity the things that cannot be changed,\\
            courage to change the things that should be changed,\\
            and the wisdom to distinguish the one from the other.

            \medspace

            Glory be to \textit{the Meatball}, to \textit{the Sauce}, and to \textit{the Holy Noodle}. \\
            As it was in the beginning, is now, and ever shall be. \\
            R'Amen.
        \end{center}
    \clearpage

%%% The following page is intentionally left as blank
% White attachment form
\hbox{ }
\thispagestyle{empty}
\clearpage
\end{document}